app/models/embed/purl/resource_file.rb

```diff
@@ -58,6 +58,10 @@ def downloadable?
           NON_DOWNLOADABLE_ROLES.exclude?(role)
       end
 
+      def viewable?
```

Comment:
Can we change the name of this?  The name `viewable?` implies  "viewable to the current user" to me.  In this case it's more like "Is not no download". I know that's a double negative, but we talk a lot about "no-download" as a specific access level.

Reply:
sounds good. Updated.